homeassistant/components/hausbus/event_handler.py

```diff
@@ -0,0 +1,19 @@
+"""Event handler abstract class."""
```

Comment:
probably belongs to the library

Reply:
This is just an abstract class/function call that we can use to set up an entity at runtime, when we receive the event from the library that a new entity was detected.